app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/vaultunlock/VaultUnlockScreen.kt

```diff
@@ -183,6 +183,7 @@ fun VaultUnlockScreen(
             ) {
                 if (!state.hideInput) {
                     BitwardenPasswordField(
+                        autoFocus = true,
```

Comment:
Do we want this to be conditional so it does not mess with the biometrics prompt that auto-appears when you have that set up?

```kotlin
autoFocus  = state.showBiometricLogin
```

Reply:
I am not seeing the auto launch of the prompt happen that consistently (could be something about the SDK creating the cipher stuff that I don't understand) with or without these changes fwiw. I think I would favor this suggestion though as if I had biometrics enabled that would likely be my preferred way to login so the text field not being focused wouldnt be a big deal or something I expected.